\hypertarget{classpip_1_1__internal_1_1exceptions_1_1PipError}{}\doxysection{pip.\+\_\+internal.\+exceptions.\+Pip\+Error Class Reference}
\label{classpip_1_1__internal_1_1exceptions_1_1PipError}\index{pip.\_internal.exceptions.PipError@{pip.\_internal.exceptions.PipError}}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Base pip exception\end{DoxyVerb}
 Inheritance diagram for pip.\+\_\+internal.\+exceptions.\+Pip\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=9.716088cm]{classpip_1_1__internal_1_1exceptions_1_1PipError}
\end{center}
\end{figure}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+internal/exceptions.\+py\end{DoxyCompactItemize}
